modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor5/plugins/ItemSelector.tsx

```diff
@@ -12,6 +12,12 @@ class ItemSelector extends Plugin {
 	init() {
 		const editor = this.editor;
 
+		const commandName = 'itemSelectorCommand';
+
+		editor.commands.add(commandName, new Command(editor));
+
+		const command = editor.commands.get(commandName)!;
+
```

Comment:
The key code we are hooking onto is 
```js
for ( const command of editor.commands.commands() ) {
	command.forceDisabled( COMMAND_FORCE_DISABLE_ID );
}
```
[here](https://github.com/ckeditor/ckeditor5/blob/v43.3.1/packages/ckeditor5-source-editing/src/sourceediting.ts#L337-L339) in `SourceEditing` plugin. That's the mechanism how it disables all controls. In the solution in this PR, we are registering a new command, binding it's enabled state to two buttons.